examples/private_dns_zones.tfvars

```diff
@@ -0,0 +1,44 @@
+private_dns_zones = {
+  storage_account_blob = {
+    resource_kind      = "storage_blob"
+    resource_group_ref = "rg_test"
+    vnet_ref           = ["vnet_test", "vnet_test2"]
```

Comment:
Rename to `vnet_refs`. Also the vnet object is not a child of the private dns zone. Rename this field to something that reflects the fact that we're linking with the vnets.